src/components/TextField/SbTextField.vue

```diff
@@ -75,31 +115,82 @@ export default {
     hasValue() {
       return this.computedValue !== null && ('' + this.computedValue).length > 0
     },
+
     hasSpecialClass() {
       return [
         this.error && 'sb-textfield__input--error',
         this.ghost && 'sb-textfield__input--ghost',
         !this.error && !this.ghost && 'sb-textfield__input--default',
       ]
     },
+
     hasIcon() {
       return [
         this.iconLeft && 'sb-textfield__input--with-icon-left',
         this.iconRight && 'sb-textfield__input--with-icon-right',
       ]
     },
+
     hasTextOnSide() {
       return [
         this.prefix && 'sb-textfield__input--with-prefix',
         this.suffix && 'sb-textfield__input--with-suffix',
       ]
     },
+
     componentClasses() {
       return [...this.hasTextOnSide, ...this.hasIcon, ...this.hasSpecialClass]
     },
+
+    isTextAreaType() {
+      return this.type === 'textarea'
+    },
+
     showClearIcon() {
       return this.hasValue && this.clearable
     },
+
+    showCounter() {
+      return this.maxlength && !this.showError
+    },
+
+    showError() {
+      return this.error && this.errorMessage
+    },
+
+    maxlengthParsed() {
+      return parseInt(this.maxlength || 0)
+    },
+
+    isRaiseMaxlength() {
+      return this.computedValueLength >= this.maxlengthParsed
+    },
+
+    isAlmostRaiseMaxlength() {
+      const limit = Math.floor(this.maxlengthParsed * 0.2)
+
+      return this.computedValueLength >= this.maxlengthParsed - limit
+    },
+
+    computedValueLength() {
+      if (typeof this.computedValue === 'string') {
```

Comment:
Não poderia fazer a coerção pra string direto? 
`return \`${this.computedValue}.length\``